test/Adapter/PhpSerializeTest.php

```diff
@@ -122,15 +122,16 @@ public function testUnserializeObject()
         $this->assertEquals($expected, $data);
     }
 
-    public function testUnserializingNonserializedStringReturnsItVerbatim()
+    public function testUnserializingNoStringRaisesException()
     {
-        $value = 'not a serialized string';
-        $this->assertEquals($value, $this->adapter->unserialize($value));
+        $value = null;
+        $this->setExpectedException('Zend\Serializer\Exception\RuntimeException');
```

Comment:
Please add the expected exception message too so there is no risk of catch any other exception of the same type.